src/gl/vglRenderer.js

```diff
@@ -249,7 +249,7 @@ geo.registerRenderer('vgl', geo.gl.vglRenderer);
   geo.gl.vglRenderer.supported = function () {
     if (checkedWebGL === undefined) {
       /* This is extracted from what Modernizr uses. */
-      var canvas, ctx, exts;
+      var canvas, ctx, exts; // eslint-disable-line no-unused-vars
```

Comment:
It's a special case.  The variables are used in the try/catch block below.  It's possible we could get rid of them, but I didn't think it worth my time figuring out.